CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 # Ewon Flexy Tag Alarm Library Changelog
 
+## v1.0.1
+### Major Changes
+- None
+### Minor Changes
+- Bugfix: Fixed exception thrown when TagInfo array has offset.
```

Comment:
I think this needs to be more descriptive. An exception being thrown is not always a bug.

Did you mean to say something like
Bugfix: Alarm events no longer generate indexOutOfBoundsException when TagInfoArray indices are offset.

Reply:
I think that is a much better description, so I'll switch to your suggestion

Reply:
changed